_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,248 @@
+- **SAML** – Provides authentication between an identity provider and a service provider. Use `-a saml`. OKTA provides the SAML Third-party authentication.
+- **No auth** – No authentication bypass authentication. Use `-a none`. It also defaults to No auth if `-a` flag is not specified.
+
+To learn more about Amazon Cognito, see [What is Amazon Cognito?](https://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html).
+### Bypass authentication option
+
+The reporting CLI tool allows you to integrate it into your own workflow or environment so that you can bypass authentication or potential security issues. For example, if you use the Reporting CLI tool within an AWS Lambda instance, no security issues would occur as long as you run the Reporting plugin in OpenSearch Dashboards. In this case, you would use "No auth" to bypass the authentication process. To specify "No Auth" use `--auth none` in your request. Lambda users should test to make sure they can bypass access to Dashboards without credentials using No Auth.  
+ To get a list of all options, see [Reporting CLI options](#reporting-cli-options).
+
+## Download and install the Reporting CLI tool
+
+You can get the Reporting CLI tool from the NPM registry or from the OpenSearch Downloads page.
+### Install the Reporting CLI from NPM
+
+To download the .tar archive and install the reporting tool with NPM, run the following command.
+
+To download the Reporting CLI .tar archive and install it, run the following command.
+
+```
+npm install opensearch-reporting-cli
+```
+
+### Install the Reporting CLI from OpenSearch.org
+
+You can download the `opensearch-reporting-cli` tool from the OpenSearch.org Downloads page, see [Command Line Tools](https://opensearch.org/downloads.html#command-line-tools).
+
+Next, run the following command to install the .tar archive.
+
+```
+npm install -g opensearch-reporting-cli.tar.gz
+```
+
+## Create and request a visualization report
+
+First, you need to get the URL for the Dashboards visualization that you want to download as an image file or PDF.
+
+To generate a visualization report, you need to specify the Dashboards URL.
+
+Open the Dashboards visualization of which you want to generate a report, and select **Share >  Permalinks > Generate link as Shapshot > Short URL > Copy link**.
+
+![Copy link]({{site.url}}{{site.baseurl}}/images/dashboards/dash-url.png)
+
+You will need to add the URL with the `-u` argument when you request the report in the CLI.
+
+### Example: Request a PNG file
+
+The following command shows how to request an OpenSearch Dashboards report in PNG format with basic authentication and send the report via SES to an email address:
+
+```
+opensearch-reporting-cli -u https://search-basic-auth2-3-b477rrluhlckrmwmtvp6k3xxsi.us-west-2.es.amazonaws.com/_dashboards/goto/069af6d6f3294421ec163b07fef91e5d?security_tenant=global -a basic -c admin:1234 -e ses -s user@amazon.com -r user@amazon.com -f png
```

Comment:
maybe `<email-id>` or some other placeholder instead of `user@amazon.com`

Reply:
ok